manuscript/230_Designing_for_composabiity_protocols.md

```diff
@@ -487,7 +487,7 @@ public class TimedSession : Session
 
 and there is no need to change any other code to get this working[^statemachine]. 
 
-Another added bonus of this situation that `Session` does not have to return anything from methods is that we are free to apply proxy and decorator patterns more freely to the `Session`. For example, we may have hidden sessions, that are not displayed/stored/sent at all, but retain the rest of the session functionality. We may implement the feature using a proxy, that forwards all messages it receives to the original, wrapped `Session` object, but discards the `DumpInto()` calls: 
+Another added bonus of this situation is that `Session` does not have to return anything from methods is that we are free to apply proxy and decorator patterns more freely to the `Session`. For example, we may have hidden sessions, that are not displayed/stored/sent at all, but retain the rest of the session functionality. We may implement the feature using a proxy, that forwards all messages it receives to the original, wrapped `Session` object, but discards the `DumpInto()` calls: 
```

Comment:
Actually, the "is" was already here, only after the "methods" word. I agree that the placement of "is" is a bit confusing, however, this doesn't fix the problem either. I suggest rewording it to:

"Another added bonus of the fact that `Session` methods don't have return types is that..."

What do you think?